src/js/claims-status/actions/index.jsx

```diff
@@ -93,9 +101,28 @@ export function getAppealsV2() {
       '/appeals_v2',
       null,
       (appeals) => dispatch(fetchAppealsSuccess(appeals)),
-      (error) => {
-        Raven.captureMessage(`vets_appeals_v2_err_get_appeals ${error.message}`);
-        return dispatch({ type: SET_APPEALS_UNAVAILABLE });
+      (response) => {
+        const status = getStatus(response);
+        const action = { type: '' };
+        switch (status) {
+          case '403':
+            action.type = USER_FORBIDDEN_ERROR;
```

Comment:
Hmm...It's a little weird to use these as both action types and availability status constants...

I'm not really sure what best practice is, though, so I won't complain. :man_shrugging:

Reply:
Yeah, that's a fair point. These will likely end up getting renamed when we split out v1 and v2.